src/tokenserver/handlers.rs

```diff
@@ -50,7 +50,15 @@ pub async fn get_tokenserver_result(
         node_type: req.node_type,
     };
 
-    Ok(HttpResponse::build(StatusCode::OK).json(result))
+    let timestamp = {
+        let start = SystemTime::now();
+        start.duration_since(UNIX_EPOCH).unwrap().as_secs()
+    };
+    Ok(HttpResponse::build(StatusCode::OK)
+        .header("X-Content-Type-Options", "nosniff")
```

Comment:
I wasn't sure how necessary this was, but it is found in responses from the Python Tokenserver, so I added it to be safe

Reply:
FYI:  It looks to be the Pyramid cornice library adding this header to the python tokenserver (I thought it was probably nginx)

Reply:
Do you think it would be helpful to keep? I just want to make sure omitting it doesn't change the behavior of clients if they've come to expect it

Reply:
Seems more the concern of nginx as opposed to the application code itself, so I'll remove it here and make a note to myself to include it there when we're ready

Reply:
... Gonna be that person and say, "Please add a comment here to indicate that this field is expected by the client and that you're creating a rule in Nginx to specify it." 

Otherwise it may become tribal knowledge.